notebooks/samples/tensorflow/prediction_auto-scaling/ai_platform_prediction_auto-scaling.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>For <code>embed()</code>, I think it makes more sense to use the (admittedly old-school) "Google APIs Client Library for Python" to send the prediction request. That way, you don't have to convert the instances dict to JSON, use gcloud to send the prediction request, and then convert the response from JSON back to a dict. Here's an example: <a href="https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/ml_engine/online_prediction/predict.py" target="_blank">https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/ml_engine/online_prediction/predict.py</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Good idea. I used the client library to simplify that section.